src/components/2022/toc-link.js

```diff
@@ -2,12 +2,27 @@ import React from "react"
 import PropTypes from "prop-types"
 import AnchorLink from "react-anchor-link-smooth-scroll"
 
+const DownArrow = () => {
+  return (
+    <svg className="cmp-toc-link__arrow" width="24" height="22" fill="none" stroke="currentColor" stroke-width="1.5" xmlns="http://www.w3.org/2000/svg"><path d="M14.768 19.42V1.656H0"/><path d="m22.57 12.213-7.802 7.8-7.8-7.8" /></svg>
```

Comment:
Getting a console error, updating to `strokeWidth` should fix

Reply:
Why is React so inconsistent on attributes with dashes?!?! 😩